apps/src/code-studio/pd/workshop_dashboard/WorkshopFormTemplate/sections/PartnerFacilitator.tsx

```diff
@@ -1,14 +1,87 @@
+import {SimpleDropdown} from '@code-dot-org/component-library/dropdown';
 import {Heading2} from '@code-dot-org/component-library/typography';
-import React, {FC} from 'react';
+import classNames from 'classnames';
+import React, {FC, useMemo} from 'react';
 
+import {MultiSelectInput} from '../components/MultiSelectInput';
 import {SectionProps} from '../types';
 
-export const PartnerFacilitator: FC<SectionProps> = ({state, handleChange}) => {
+import commonStyles from '../styles.module.scss';
+
+export const PartnerFacilitator: FC<SectionProps> = ({
+  config: {
+    fields: {regional_partner_id, facilitators},
+  },
+  state,
+  handleChange,
+  regionalPartners,
+  facilitators: fetchedFacilitators,
```

Comment:
What is the difference between the config `facilitators` and the `fetchedFacilitators`?